### ISSUE TITLE
Some part of guesthelper.exe is not working

### ISSUE THREAD

**Issue by didriklundberg** [guancio/kth-on-rpi2#2]
guesthelper.exe causes a problem when trying to compile with the minimal guest. This needs to be fixed if you want to use that for testing.

**Comment by guancio** [comment]
I think that for now we can just use the guest dtest

**Comment by didriklundberg** [comment]
Yep - that is exactly why I made this an Issue, so that we can forget about it until later ;)
